v1.1/admin_ui_cluster_overview.md

```diff
@@ -0,0 +1,71 @@
+
+In the node view, the graph displays the number of range replicas on the selected node.
+
+In the cluster view, the graph displays the number of range replicas on each node in the cluster. 
+
+Ranges are subsets of your data, which are replicated to ensure survivability. Ranges are replicated to a configurable number of CockroachDB nodes. 
+
+By default, the cluster-wide replication zone is set to replicate data to any three nodes in your cluster, with ranges in each replica splitting once they get larger than 64 MiB. The replicas are balanced evenly across the nodes. You can [configure replication zones](configure-replication-zones.html) to set the number and location of replicas. You can monitor the changes in configuration using the Admin UI, as described in [Fault tolerance and recovery](demo-fault-tolerance-and-recovery.html).
+
+For more information on replicas, see [Replication](high-availability.html#replication), or read the [CockroachDB design document](https://github.com/cockroachdb/cockroach/blob/master/docs/design.md#architecture).
+
+## Capacity
+<img src="{{ 'images/admin_ui_capacity.png' | relative_url }}" alt="CockroachDB Admin UI Capacity graph" style="border:1px solid #eee;max-width:100%" />
+
+In the node view, the graph displays the maximum allocated as well as available storage capacity for CockroachDB for the selected node.
+
+In the cluster view, the graph displays the maximum allocated as well as available storage capacity for CockroachDB across all nodes in the cluster.
+
+You can monitor this graph to determine when additional storage needs to be provided. 
+
+On hovering over the graph, you can see the Capacity and Available values:
+
+Metric | Description
+--------|----
+Capacity | The maximum storage capacity allocated to CockroachDB.
+Available | The free storage capacity available to CockroachDB.
+
+{{site.data.alerts.callout_info}}If you are running multiple nodes on a single machine (not recommended), and haven't specified the maximum allocated storage capacity for each node using the --store flag, then the available capacity displayed in the graph is incorrect. This is because when multiple nodes are running on a single machine, the hard disk of the machine is considered as available store for each node. The total available capacity is then displayed as the hard disk size multiplied by the number of nodes on the machine, while in reality, only one hard disk is available for all nodes. {{site.data.alerts.end}}
+
+You can configure the maximum allocated storage capacity for CockroachDB using the --store flag. For more information, see [Start a Node](start-a-node.html#store).
+
+{{site.data.alerts.callout_info}}The CockroachDB Admin UI has three additional dashboards: Distributed, Queues, and Slow Requests. These dashboards are important for CockroachDB developers. For monitoring CockroachDB, it is sufficient to monitor the Overview, Runtime, SQL, Storage, and Replication dashboards.{{site.data.alerts.end}}
```

Comment:
After recent reorganization this note now seems a bit out of place here.

Reply:
@couchand Umm..not sure why the note seems out of place now? Can you elaborate?

Reply:
Just that it is at the end of the listing of the specific metrics on the overview page.  I think it should be next to the listing of various dashboards.